nomad/client_fs_endpoint_test.go

```diff
@@ -868,7 +868,7 @@ func TestClientFS_Streaming_Local_Follow(t *testing.T) {
 		Name:   "web",
 		Driver: "mock_driver",
 		Config: map[string]interface{}{
-			"run_for":                2 * time.Second,
+			"run_for":                3 * time.Second,
```

Comment:
Didn't you change this to a string above?
```suggestion
			"run_for":                "3s"
```

Reply:
I made the duration accept both, a duration string or a simple duration (as they will get serialized as string of nanoseconds) - but I can change all references to string to avoid this confusion and allow reuse later one maybe.

Reply:
I updated all tests to only accept duration strings to avoid confusion and bring tests closer to mimic user input.

Reply:
Ah! I see. I think supporting the string path is sufficient but no need to change it now.